saleor/plugins/webhook/tests/subscription_webhooks/filterable_webhooks/test_order_cancelled.py

```diff
@@ -0,0 +1,227 @@
+import json
+from unittest.mock import patch
+
+import graphene
+from django.test import override_settings
+
+from ......core.models import EventDelivery
+from ......graphql.webhook.subscription_query import SubscriptionQuery
+from ......webhook.event_types import WebhookEventAsyncType
+from .....manager import get_plugins_manager
+
+ORDER_CANCELLED_SUBSCRIPTION = """
+subscription {
+  orderCancelled(channels: ["default-channel"]) {
+    order {
+      id
+      number
+      lines {
+        id
+        variant {
+          id
+        }
+      }
+    }
+  }
+}
+"""
+
+
+@patch(
+    "saleor.webhook.transport.asynchronous.transport.send_webhook_request_async.apply_async"
+)
+@override_settings(PLUGINS=["saleor.plugins.webhook.plugin.WebhookPlugin"])
+def test_order_cancelled(mocked_async, order_line, subscription_webhook):
+    # given
+    manager = get_plugins_manager(False)
+    order = order_line.order
+    channel = order.channel
+    channel.slug = "default-channel"
+    channel.save()
```

Comment:
This action is not required. `order` fixture uses `channel_USD` with `DEFAULT_CHANNEL_SLUG` as a slug.  It should be fixed in all tests. 
```
def channel_USD(db):
    slug = settings.DEFAULT_CHANNEL_SLUG
``` 
https://github.com/saleor/saleor/blob/64d413d87b7e3f1b7732956e966cddb9721608d1/saleor/tests/fixtures.py#L1418

Reply:
yes, I know, but wanted to do this explicitly, to highlight the slug of the channel :)

Reply:
If you want to have it explicate add assert instead of save. (Save makes an extra call to the database.)

Reply:
assert sounds good